src/foundry/common/utils/event-emitter.d.mts

```diff
@@ -52,6 +49,10 @@ declare class EventEmitter {
   dispatchEvent(event: Event): boolean;
 }
 
+declare namespace EventEmitter {
+  type EmittedEventListener = (event: Event) => any;
+}
```

Comment:
The return type should be `void`. Foundry documents it as `any` but they simply ignore the return value; this is the purpose of `void`.